server.js

```diff
@@ -1,22 +1,16 @@
-const consola = require("consola");
-const fs = require("fs");
-
-const generator = require("./generator");
+const jsonServer = require('json-server');
+const customRouter = require('./routes');
+const consola = require('consola');
 
 consola.LogLevel = 4;
+const server = jsonServer.create();
+const middlewares = jsonServer.defaults();
 
-const main = () => {
-  const mode = "dev";
-  const data = generator(mode);
-
-  consola.success("Generation completed. Populating latest json into db.json");
+const db = process.env.NODE_ENV === 'development' ? 'dev_db.json' : 'db.json';
+const router = jsonServer.router(db);
 
-  fs.writeFile(
-    mode === "dev" ? "dev_db.json" : "db.json",
-    JSON.stringify(data),
-    "utf8",
-    () => consola.success("db.json populated!")
-  );
-};
+server.use(middlewares);
+server.use(customRouter);
+server.use(router);
```

Comment:
Custom routes are given priority. If there's a clash, they get overridden by the custom route. 

And, I did not add custom routes for now, because this PR focuses on making use of live-server programmatically. The routes will be handled in a separate PR.

Reply:
hmm, okayyy. But not embedding comments along with users and stuff may result in breaking change

Reply:
Yeah, that's something we need to take care of when we are building. But that's really not a focus in this PR. I'll mark this as resolved. Do open it back if you want to talk more about it.